plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetViewer.java

```diff
@@ -2944,7 +3061,15 @@ public void run() {
         viewMenu.add(new Separator());
         if (model.getDocumentAttribute() == null) {
             if (valueController != null) {
-                viewMenu.add(new SetRowColorAction(this, attr, valueController.getValue()));
+                String bindingText = attr.getName() 
+                        + " = "
+                        + UITextUtils.getShortText(this.getControl(),
+                        CommonUtils.toString(valueController.getValue()), 100);
```

Comment:
We use 4-spaces indentation always.
```suggestion
                String bindingText = attr.getName() 
                    + " = "
                    + UITextUtils.getShortText(this.getControl(),
                    CommonUtils.toString(valueController.getValue()), 100);
```